### ISSUE TITLE
When using Power Code, my entire code is experiencing jitter.

### ISSUE THREAD

**Issue by Tiro-Wang** [hoovercj/vscode-power-mode#122]
Can we prevent it from moving?

**Comment by Smalldy** [comment]
IT IS POWER!!!!!!!!!

**Comment by LouisDvr** [comment]
@Tiro-Wang Yes you can, you just need to turn it off in the settings (uncheck the box)

<img width="445" alt="Image" src="https://github.com/user-attachments/assets/62ed3fa1-047a-43fb-be10-6b72f340c55b" />
